diktat-rules/src/test/resources/test/paragraph3/enum_separated/LastElementCommentTest.kt

```diff
@@ -0,0 +1,25 @@
+package test.paragraph3.enum_separated
+
+enum class Foo1 {
+    A,
+    B,
+    C // some commnet
+}
+
+enum class Foo2 {
+    A,
+    B,
+    C// some commnet
+}
+
+enum class Foo3 {
+    A,
+    B,
+    C /* some commnet */
+}
+
+enum class Foo4 {
+    A,
+    B,
+    C/* some commnet */
```

Comment:
it's not affected right now and support it requires some changes.
`appendNewlineMergingWhiteSpace` doesn't work for such case because KDOC is another node and doesn't have a common parent with ENUM_ELEMENT

Reply:
OK, I thought that KDocs might be represented differently in the tree. Then the current version is fine.

Reply:
actually, I supported. please check (added tests for such cases)